### DISCUSSION TITLE
Server port and exposed port must be the same

### DISCUSSION THREAD

**Discussion by rasmusson** [itzg/docker-minecraft-bedrock-server#393] [Show and tell]
I had some problems getting the server up and running on kubernetes using node port. From the constrainer I exposed the default server port 19132 but I exposed in on node port 30008. Im able to the make a first connection to the server. But I always ended up with "Unable to connect to world". 
Turns out the the client and server uses the port you set in the client app server config to send ping and pong UDP messages back and forth. The ping answer from the server contains the port the at the server was configurated to use using SERVER_PORT. If this is not the same as the one you configured in the client app server config, the client app still tries to contact the port from the ping response, and causing this error. 

So in conclusion. What ever port you expose to the client, must be the same as the one the server is configured to run on. Be it the default 19132 or a custom, specified with SERVER_PORT environmental variable
